src/frontend/apps/e2e/__tests__/app-impress/doc-tree.spec.ts

```diff
@@ -215,43 +213,38 @@ test.describe('Doc Tree', () => {
 
     await verifyDocName(page, docParent);
 
-    await page.getByRole('button', { name: 'Share' }).click();
-
-    await addNewMember(page, 0, 'Owner', 'impress');
-
-    const list = page.getByTestId('doc-share-quick-search');
-    const currentUser = list.getByTestId(
-      `doc-share-member-row-user@${browserName}.test`,
-    );
-    const currentUserRole = currentUser.getByLabel('doc-role-dropdown');
-    await currentUserRole.click();
-    await page.getByLabel('Administrator').click();
-    await list.click();
-
-    await page.getByRole('button', { name: 'Ok' }).click();
-
```

Comment:
This part should not be removed, we are not asserting that only a owner can detach a document anymore.